Chapter_8/Chapter_8. Exercises/Chapter_8. Exercises/cont.cpp

```diff
@@ -0,0 +1,148 @@
+#include "head.h"
+
+void print_ex2(const string& comment, const vector<int>& v) 
+{
+    cout << comment << ": ";
+    for (int num : v) 
+    {
+        cout << num << " ";
+    }
+}
+
+vector<int> fibonacci_ex3(int x, int y, vector<int> v, int n) 
+{
+    v.push_back(x);
+    v.push_back(y);
+    if (n > 1)
+    {
+        for (int i = 2; i < n; ++i)
+        {
+            v.push_back(v[i - 1] + v[i - 2]);
+        }
+    }
```

Comment:
Условие `if (n > 1)` излишне: если мы его опустим, всё продолжит работать так же, как и без него
```suggestion
    for (int i = 2; i < n; ++i)
    {
        v.push_back(v[i - 1] + v[i - 2]);
    }
```